task/06-conditions-n-loops-tasks.js

```diff
@@ -166,7 +187,11 @@ function isInsideCircle(circle, point) {
  *   'entente' => null
  */
 function findFirstSingleChar(str) {
-    throw new Error('Not implemented');
+    for (let i = 0; i < str.length; i++) {
+        let letter = str.charAt(i);
+        if (str.indexOf(letter) == i && str.indexOf(letter, i + 1) == -1) return letter;
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
""0"" == 0
and ""0"" === 0)"